src/settings-ui/Settings.UI/Views/PowerAccentPage.xaml

```diff
@@ -66,6 +90,11 @@
                                             LargeChange="100"/>
                         </controls:Setting.ActionContent>
                     </controls:Setting>
+                    <!--<controls:Setting x:Uid="QuickAccent_DisableFullscreen" Icon="&#xE916;">
+                        <controls:Setting.ActionContent>
+                            <ToggleSwitch IsOn="{x:Bind ViewModel.DisableFullscreen, Mode=TwoWay}" x:Uid="DisableFullscreen" HorizontalAlignment="Right"/>
```

Comment:
ToggleSwitchs are used for immediate action, this should be a checkbox
https://learn.microsoft.com/windows/apps/design/controls/toggles#choosing-between-toggle-switch-and-check-box

Reply:
I think fullscreen detection should come as part of a different PR.

Reply:
Agree. I removed them.